test/parallel/test-http-early-hints-multiple-links.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+require('../common');
+const assert = require('assert');
+const http = require('http');
+const debug = require('util').debuglog('test');
+
+const testResBody = 'response content\n';
+
+const server = http.createServer((req, res) => {
```

Comment:
This and all of the other callbacks need the appropriate `common.mustCall()` (and possibly `common.mustNotCall()`) wrappers.

Reply:
updated, thanks for the review!